### PR TITLE
fix(drawer): screenshots use list

### PR DESCRIPTION
**Pull request by moog16**
fixes https://github.com/material-components/material-components-web-react/issues/364

### CODE REVIEW

File: test/screenshot/drawer/List.js [hunk 1]
```diff
@@ -1,59 +1,96 @@
...
-  const bottomItems = [{
-    title: 'Settings', icon: 'settings',
-  }, {
-    title: 'Help & feedback', icon: 'announcement',
-  }];
 
-  return (
-    <nav className='mdc-list'>
-      {topItems.map(renderListItem)}
+// const renderListItem = ({
```
Comment (bonniezhou) [inline test/screenshot/drawer/List.js]: delete comments

File: test/screenshot/drawer/List.js [hunk 2]
```diff
@@ -1,59 +1,96 @@
...
+    selectedIndex: 1,
+  };
 
-      <hr className='mdc-list-divider' />
-      {bottomItems.map(renderListItem)}
-    </nav>
-  );
+  render() {
+    const topItems = [{
+      title: 'Inbox', icon: 'inbox', activated: true,
```
Comment (bonniezhou) [inline test/screenshot/drawer/List.js]: `activated` is unused

File: test/screenshot/drawer/List.js [hunk 3]
```diff
@@ -1,59 +1,96 @@
...
+} from '../../../packages/list/index';
+import uuidv1 from 'uuid/v1';
 
-const renderListItem = ({
-  title, icon, activated, // eslint-disable-line react/prop-types
-}, index) => {
+import '../../../packages/list/index.scss';
+
+
+const renderListItem = ({title, icon, activated}) => {
```
Comment (bonniezhou) [inline test/screenshot/drawer/List.js]: `activated` is unused

**Comment by moog16** [comment]
Closing as its almost 6 months

**Pull request closed without merging**